### PR TITLE
Update origin trial status for Soft Navigations

### PR DESCRIPTION
**Pull request by tunetheweb**
Changes proposed in this pull request:

- Updates latest status of origin trial

### CODE REVIEW

File: site/en/blog/soft-navigations-experiment/index.md [hunk 1]
```diff
@@ -64,10 +64,14 @@ Soft navigations are not enabled by default in Chrome, but are available for exp
...
 
 Site owners can choose to include the origin trial on their pages for all, or for just a subset of users. Be aware of the [implications section](#what-are-the-implications-of-enabling-soft-navigations-in-chrome) above as to how this changes how your metrics may be reported, especially if enabling this origin trial for a large proportion of your users. Note that CrUX will continue to report the metrics in the existing manner regardless of this soft navigation setting so is not impacted by those implications. It should also be noted that origin trials are also limited to enabling experimental features on a maximum of 0.5% of all Chrome page loads as a median over 14 days, but this should only be an issue for very large sites.
 
+{% Aside 'important' %}
+The Soft Navigations origin trial has now ended. The Chrome team are taking onboard the feedback received and will be making some adjustments and are aiming to relaunch a second origin trial in Q3 2023 for further feedback.
```
Comment (yoavweiss) [inline site/en/blog/soft-navigations-experiment/index.md:72]: end of Q3?

File: site/en/blog/soft-navigations-experiment/index.md [hunk 2]
```diff
@@ -3,12 +3,12 @@
 layout: 'layouts/blog-post.njk'
 title: Experimenting with measuring soft navigations
 description: |
- The Chrome team is working on better measuring so-called soft navigations used by Single Page Applications and a new API is now in origin trial to allow sites to experiment with this too.
+ The Chrome team is working on better measuring so-called soft navigations used by Single Page Applications and a new API is now available to allow sites to experiment with this too.
```
Comment (yoavweiss) [inline site/en/blog/soft-navigations-experiment/index.md:6]: "available" may confuse folks to think it shipped?

File: site/en/blog/soft-navigations-experiment/index.md (hunk 1 above)
Comment (tunetheweb) [reply site/en/blog/soft-navigations-experiment/index.md:72]: Will I just say Q4? Or "later in the year"?

File: site/en/blog/soft-navigations-experiment/index.md (hunk 2 above)
Comment (tunetheweb) [reply site/en/blog/soft-navigations-experiment/index.md:6]: WDYT about this then:

```suggestion
 The Chrome team is working on better measuring so-called soft navigations used by Single Page Applications and a new API is now available behind a flag to allow sites to experiment with this too.
```

Comment (yoavweiss) [reply site/en/blog/soft-navigations-experiment/index.md:6]: yes!

File: site/en/blog/soft-navigations-experiment/index.md (hunk 1 above)
Comment (tunetheweb) [reply site/en/blog/soft-navigations-experiment/index.md:72]:
```suggestion
The Soft Navigations origin trial has now ended. The Chrome team are taking onboard the feedback received and will be making some adjustments and are aiming to relaunch a second origin trial from Chrome 117 (due for release in September 2023) for further feedback.
```

**Review by malchata — APPROVED** [review]

**Pull request merged**